src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -212,16 +464,22 @@ EscrowCreate::doApply()
         return tefINTERNAL;
 
     // Check reserve and funds availability
-    {
-        auto const balance = STAmount((*sle)[sfBalance]).xrp();
-        auto const reserve =
-            ctx_.view().fees().accountReserve((*sle)[sfOwnerCount] + 1);
+    STAmount const amount{ctx_.tx[sfAmount]};
```

Comment:
`doApply` function has become too big. Please consider to move out some part of it to helper functions.